exercise_03/generateRandomNumberInRange.js

```diff
@@ -3,3 +3,14 @@
  * input: number (limit)
  * output: number
  */
+
+/**
+ * Generate a rounded random number within the specified range.
+ * @param {number} limit - The upper limit for the random number.
+ * @returns {number} - A rounded random number within the specified range.
+ */
+function generateRandomNumberInRange(limit) {
+  return Math.round(Math.random() * limit);
+}
+
+module.exports = generateRandomNumberInRange;
```

Comment:
Don't use commonJs modules (`module.exports`) 
```suggestion
export default generateRandomNumberInRange;
```